client/js/entities/model/bioticState.js

```diff
@@ -0,0 +1,39 @@
+define(function(require){
+
+var EntityState = require("./entityState");
+var Health = require("./health");
+
+return function BioticState(entityState, health) {
+	var self = this;
+
+	self.entityState = entityState || new EntityState("nil", null, 0);
```

Comment:
Doesn't mean anything. It's just a human readable string indicating that this particular thing doesn't have a real value. Chose nil just because "null" and "0" make the other two. Yay for diversity!